lib/device_api/android.rb

```diff
@@ -10,6 +10,7 @@
 
 # Load additional device types
 require 'device_api/android/device/kindle'
+require 'device_api/android/device/samsung'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.